CONTRIBUTING.md

```diff
@@ -68,38 +69,97 @@ Please see `./ci/features`.
 To test the usage of the Conjur Service Broker within a CF deployment, you can
 follow the demo scripts in the [Cloud Foundry demo repo](https://github.com/conjurinc/cloudfoundry-conjur-demo).
 
-To run the test suite on your local machine:
+#### Running the Dev Environment
+To test your changes within a running instance of Cloud Foundry and Conjur, run:
+
+```shell script
+./ci/start_dev_environment
+```
+
+This starts Conjur and Cloud Foundry containers, and provides terminal 
+access to the Cloud Foundry container. You do not need to restart the container
+after you make changes to the project.
+
+To run the `cucumber` within the development environment, run the following command from
+the `ci` directory, within the container:
+
+```shell script
+cucumber \
+    --format pretty \
+    --format junit \
+    --out ./features/reports \
+    --tags 'not @integration'
 ```
-$ ./package.sh   # Create the conjur-env binary in the vendor dir and a ZIP of the project contents
-$ ./test.sh      # Run the test suite
+
+#### Setup
```

Comment:
Maybe add an explanation of what this "Setup" is for in the heading?

Reply:
Added this note to the prerequisites, what do you think?
```
The following prerequisites and all sections below pertain to building and running the Conjur Buildpack locally,
unless otherwise specified.
```

Reply:
@BradleyBoutcher Apologies for the noise, but yeah, this is good, even if you want to delete the note in the prerequisites. I think what was throwing me off is that the `Setup` section isn't the first section (as in "Wait, you're doing Setup now, partway through the process?"). The explanation that you have below the `Setup` head 'splains it well enough.